File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter06/JAVA/FuncCircle.java
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 06.-Subprogramas (subalgoritmos): Funciones
    Problema 11.-Se trata de realizar el cálculo del área de un círculo y la longitud de la circunferencia en función del valor del radio
                 leído desde el teclado.
    
    Ejemplo 11
    Pagina 223-224
*/
import java.util.Scanner;

public class FuncCircle{
    static double calArea(float radio){
        return Math.PI*radio*radio;
    }

    static double calcCircun(float radio){
        return Math.PI*2*radio;
    }

    public static void main(String[] args){
        Scanner in=new Scanner(System.in);
        float radio=0.0F;

        System.out.print("Programa que calcula el area de un circulo y la longitud de la circunferencia\nIngrese el radio del circulo: ");
        radio=in.nextFloat();

        System.out.print("\nArea: "+calArea(radio)+"\nCircunferencia: "+calcCircun(radio)+"\n\n");
    }    
}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter06/JAVASCRIPT/JS/funcCircle.js
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 06.-Subprogramas (subalgoritmos): Funciones
    Problema 11.-Se trata de realizar el cálculo del área de un círculo y la longitud de la circunferencia en función del valor del radio
                 leído desde el teclado.
    
    Ejemplo 11
    Pagina 223-224
*/
function calArea(radio){
    return Math.PI*radio*radio;
}

function calcCircun(radio){
    return Math.PI*2*radio;
}

function calculate(){
    var radio;
    var varSpan=document.createElement("span");
    varSpan.setAttribute("id", "idSpan");
    
    if(document.getElementById("idBtnCalculate").value=="Calcular"){

        radio=document.getElementById("idNum").value;
        
        document.getElementById("idResult").value=varSpan.innerHTML+="<br>Area: "+calArea(radio)+"<br>Circunferencia: "+calcCircun(radio)+"<br><br>";
        
        document.getElementById("idResult").append(varSpan);
        document.getElementById("idBtnCalculate").value="Limpiar";
    }else{
        cleanPage();
    }
}        
            
function cleanPage(){
    document.getElementById("idNum").value="";
    document.getElementById("idSpan").remove();
    document.getElementById("idBtnCalculate").value="Calcular";
}   